README.md

```diff
@@ -24,7 +24,7 @@ Please visit the [documentation].
 | **C**                         | C99, C11             | GCC 7+, clang 5+, MSVC 2017+ | Host- and VM-side |
 | **C++**                       | C++17                | GCC 7+, clang 5+, MSVC 2017+ | Host- and VM-side |
 | **Go** _(bindings)_           | 1.11+ (with modules) |                              | Host-side only    |
-| **Rust** _(bindings)_[¹](#n1) | 2018 edition         | 1.37.0 and newer             | VM-side only      |
+| **Rust** _(bindings)_[¹](#n1) | 2018 edition         | 1.47.0 and newer             | VM-side only      |
```

Comment:
Let's do this considering 1.4.7 was released in 2020: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1470-2020-10-08